tests/framework/session/test_session.py

```diff
@@ -285,6 +286,18 @@ def test_create(
         assert session.load_context() is mock_context_class.return_value
         assert isinstance(session._get_config_loader(), ConfigLoader)
 
+    @pytest.mark.usefixtures("mock_settings")
+    def test_create_session_with_unclosed_session(
+        self, fake_project, mock_package_name
+    ):
+        first_session = KedroSession.create(mock_package_name, fake_project)
+        _activate_session(first_session)
+        assert kedro.framework.session.session._active_session is first_session
+
+        new_session = KedroSession.create(mock_package_name, fake_project)
+        _activate_session(new_session)
+        assert kedro.framework.session.session._active_session is new_session
+
```

Comment:
Would it be possible to do the context manager of this just to check it works the same way?
```
with KedroSession.create() as first_session:
    assert kedro.framework.session.session._active_session is first_session
    with KedroSession.create() as new_session:
        assert kedro.framework.session.session._active_session is new_session
    assert kedro.framework.session.session._active_session is new_session ## is that right?
```

Reply:
I think it makes sense to check that it's now possible to create two sessions without it failing, like it did previously.